rfcs/20200106-prelu/README.md

```diff
@@ -0,0 +1,76 @@
+# Introducing PReLu support in OneDnn
+
+## Introduction
+
+This paper https://arxiv.org/abs/1502.01852 introduced PReLU activation function.
+Request comes from Caffe users: "PRelu is used in some POR topologies, such as
+VNET, SRGAN. It takes most of time if the original implementation, is not good
+optimized. For example VNET's train perf with latest Intel Caffe is 2.4s, PReLU
+(has optimized by openmp) spends 429ms (not including reorder time), 17% of total
+time. If we uses MKLDNN ReLU to make projection of the perf, it will come to
+~1.8s." Some other data scientist in discussion mentioned that he encouter PRelu
+issue when he tried to optimize the inference pipeline of a popular face-related
+application - PRelu tooks 47 % of time in his application.
+
+Currently in oneDnn we support ReLu activation function in eltwise operation:
+https://oneapi-src.github.io/oneDNN/dev_guide_eltwise.html. LeakyReLu can be
+achieved by specifying non zero alpha parameter. PReLu is specific variant of
+LeakyRelu, where alpha is learnable parameter. It cames with two flavours:
+- channel-wise PReLU - where number of trained alpha parameters is equal to
+channel number
+- channel-shared PReLu - where there is only one trainable alpha
```

Comment:
style, throughout: use a consistent capitalization scheme for "prelu" (probably PReLU is best)